pullrequest.rst

```diff
@@ -166,6 +166,11 @@ any discussion of what the pull request is trying to solve (e.g. fixing a
 spelling mistake), then the pull request needs to have the "trivial" label
 added to it.
 
+If your pull request involves several commits as a result of addressing code
+review comments, please do not squash them.  When you squash your commits,
```

Comment:
Thanks @pfmoore and @bitdancer. I updated this. Please check it this sounds better.